src/main/java/model/User.java

```diff
@@ -0,0 +1,70 @@
+package model;
+
+public class User {
+    private int userId;
+    private String firstName;
+    private String lastName;
+    private int mobileNumber;
+    private String userInformation;
+    private String email;
+    private String password;
+
+    public User() {
+
+    }
+
+    public User(int userId, String firstName, String lastName, int mobileNumber, String userInformation, String email, String password) {
```

Comment:
Have only mandatory attributes to initialize the object here. Intialize the rest with a builder.